packages/client/src/pages/CheckerPage.tsx

```diff
@@ -158,12 +172,15 @@ const CheckerPage = () => {
   };
 
   // On LocationSubmit
-  const handleNewAddressSubmit = (address: any) => {
+  const handleNewAddressSubmit = (address: Address) => {
     setTopicData({
       activeComponents: [sections.QUESTIONS],
       finishedComponents: [sections.LOCATION_INPUT],
       address,
     });
+
+    // Autofill `checker` when `address` is submitted
+    checker.autofill(autofillResolvers, { address });
```

Comment:
Since `checker` is initialised before submitting the address we need to manually call `autofill`